package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./build/main.js",
   "productName": "Jitsi Meet",
   "scripts": {
-    "start": "webpack --config ./webpack.main.js --mode development && concurrently \"npm:watch\" \"electron ./build/main.js\"", 
+    "start": "npm run build && electron ./build/main.js",
```

Comment:
Why are you changing this? It's very useful to have hot-reloading while developing!

Reply:
By mistake :( sorry!

Reply:
Fixed!